mmv1/products/notebooks/api.yaml

```diff
@@ -146,6 +146,8 @@ objects:
     create_url: projects/{{project}}/locations/{{location}}/instances?instanceId={{name}}
     self_link: projects/{{project}}/locations/{{location}}/instances/{{name}}
     create_verb: :POST
+    update_verb: :PATCH
+    update_mask: true
     input: true
```

Comment:
Looks like we have `input: true` for this resource which means the resource can not be undated. I wonder if the update functionality works for this resource after adding `PATCH`. Can we also add a test?

Reply:
Why did we add those lines? This PR is supposed to be for `Runtime` not `Instance`.

Reply:
I agree. We can exclude these two lines from this PR. I just saw these two lines added in the PR and if that is intentionally, I would suggest that we add an update test for it.